tests/unit/routes/test_link_router.py

```diff
@@ -136,103 +134,6 @@ def test_link_success(self, patched_subprocess, basic_algorithm, patients, clien
         assert response_6.json()["prediction"] == "no_match"
         assert not response_6.json()["results"]
 
-    @mock.patch("recordlinker.database.algorithm_service.get_algorithm")
-    def test_link_enhanced_algorithm(
```

Comment:
any specific reason for deleting this rather than just renaming it to use the default algorithm?

Reply:
Yes! The test right above it, `test_link_success`, was originally setup to run all of this code with the dibbs basic algorithm. Since we no longer have that algorithm defined, these tests are now duplicates of each other with just different names. So just reducing the test scope down to one test that covers all of the six matches we want.